uiowa/uiowa01
# Feature: Layout builder active condition plugin

This update add a condition plugin - which can be set as a visibility setting in a block - that determines whether layout builder is active or not. This allows us to replace the usage of the content type visibility setting for content types that had layout builder overrides enabled. It also allows the removal of all the code that was in place to try to ensure that the right visibility settings were active for the relevant content types, which was confusing and may not have been working correctly.

# How to Test
1. `drush @theming.local si collegiate`
2. Create a basic page item. Check that the page title block (in the content region) doesn't show up.
3. Enable the entity_type_events module
4. Create an event item. Ensure that the page title block (in the content region) *does* show up.
5. View the /events page and ensure that the page title block does show up.
